<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Swiftbot 1.0.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>658</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="leftPanel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="accountsTreeWidget">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="indentation">
           <number>10</number>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="quickActionButtons">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(15,15, 15);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="accountManagerButton">
             <property name="toolTip">
              <string>Gestionnaire de comptes</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/icons/icons/group_gear.png</normaloff>:/icons/icons/group_gear.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="quickActionButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>38</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="toolTip">
              <string>Action rapide</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton::menu-indicator{
    image: none;
}</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/icons/icons/gear_in.png</normaloff>:/icons/icons/gear_in.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="popupMode">
              <enum>QToolButton::InstantPopup</enum>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="mainPanel">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="mainPanelLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionGestionnaire_de_comptes"/>
    <addaction name="actionCr_er_un_personnage"/>
    <addaction name="separator"/>
    <addaction name="actionLancer_en_MITM"/>
    <addaction name="actionLancer_Dofus_Sans_MITM"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QMenu" name="menuInformations">
    <property name="title">
     <string>Informations</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOptions"/>
   <addaction name="menuInformations"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGestionnaire_de_comptes">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/group_gear.png</normaloff>:/icons/icons/group_gear.png</iconset>
   </property>
   <property name="text">
    <string>Gestionnaire de comptes</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Gestionnaire de comptes&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Le gestionnaire de comptes vous permet de configurer vos comptes mais aussi de cr√©er des personnages.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionCr_er_un_personnage">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/user_add.png</normaloff>:/icons/icons/user_add.png</iconset>
   </property>
   <property name="text">
    <string>Cr√©er un personnage</string>
   </property>
  </action>
  <action name="actionLancer_en_MITM">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/console.png</normaloff>:/icons/icons/console.png</iconset>
   </property>
   <property name="text">
    <string>Connexion MITM</string>
   </property>
  </action>
  <action name="actionLancer_Dofus_Sans_MITM">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/dofus.png</normaloff>:/icons/icons/dofus.png</iconset>
   </property>
   <property name="text">
    <string>Lancer Dofus (Sans MITM)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>accountsTreeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectedAccount(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>673</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accountManagerButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAccountManager()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGestionnaire_de_comptes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAccountManager()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedAccount(QTreeWidgetItem*,int)</slot>
  <slot>openAccountManager()</slot>
 </slots>
</ui>
